src/components/search/character-check.js

```diff
@@ -24,43 +26,46 @@ export default class CharCheck {
   updateCheckReadout(event, firstRun) {
     const value = this.input.value;
     const remaining = this.checkVal - value.length;
+    const noMaxCheck = this.checkMinOnly;
     // Prevent aria live announcement when component initialises
     if (!firstRun && event.inputType) {
       this.checkElement.setAttribute('aria-live', 'polite');
     } else {
       this.checkElement.removeAttribute('aria-live');
     }
 
-    this.checkRemaining(remaining);
+    this.checkRemaining(remaining, noMaxCheck);
```

Comment:
We don't need to pass it to the function. We can just check `this.checkMinOnly` at the point we need to as it doesn't change after the class is instantiated.